frontend/README.md

```diff
@@ -30,9 +30,12 @@ The build is minified and the filenames include the hashes.\
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
+### `npm run e2e`
+
+Runs all Cypress tests. Requires having already run `npm start` in frontend, and also backend once tests involving database access have been written. (Temporary, writing a script to do the set up steps beforehand currently)
```

Comment:
Will need to make an issue to refactor frontend readme once backend has been connected

Reply:
Yeah I might just give that job to the next team. Will edit that sentence